recipes/hcrystalball/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "hcrystalball" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 877f9070e04ec08824b192f7fa45266454bead58efac1616ba8fe89a0b881962
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - numpy
+    - scipy
+  run:
+    - python >=3.7
+    - {{ pin_compatible('numpy') }}
```

Are you building against the numpy C API? I'm guessing not, if so:
```suggestion
    - numpy
```